security/grub-extend-pcr.py

```diff
@@ -91,3 +91,21 @@ def test_tsseventextend(self):
             pcr8_flag = pcr9_flag = False
         if not (pcr8_flag and pcr9_flag):
             self.fail("PCR 8 and/or PCR 9 not having correct values.")
+
+    def test_boot_aggregate(self):
+        '''
+        Output validation for boot aggregate from two different files
+        ascii_runtime_measurements and binary_bios_measurements
+        '''
+        if not os.path.exists("/sys/kernel/security/ima/ascii_runtime_measurements"):
+            self.fail("files doesn't exist")
+        if not os.path.exists("/sys/kernel/security/tpm0/binary_bios_measurements"):
+            self.fail("file doesn't exist")
+        cmd1 = "head -n1 /sys/kernel/security/ima/ascii_runtime_measurements"
```

Comment:
Please use the variable name for the filename(s). You used the filename more than once.

Reply:
resolved